protocol/app/vote_extensions/oracle_client.go

```diff
@@ -0,0 +1,85 @@
+package vote_extensions
+
+import (
+	"context"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	priceskeeper "github.com/dydxprotocol/v4-chain/protocol/x/prices/keeper"
+	"github.com/dydxprotocol/v4-chain/protocol/x/prices/types"
+	oracleclient "github.com/skip-mev/slinky/service/clients/oracle"
+	oracletypes "github.com/skip-mev/slinky/service/servers/oracle/types"
+	types2 "github.com/skip-mev/slinky/x/oracle/types"
+	"google.golang.org/grpc"
+	"strconv"
+)
+
+type OracleClient struct {
+	Slinky       oracleclient.OracleClient
+	PricesKeeper priceskeeper.Keeper
+	Context      *sdk.Context
+}
+
+func (o OracleClient) Start(ctx context.Context) error {
+	return o.Slinky.Start(ctx)
+}
+
+func (o OracleClient) Stop() error {
+	return o.Slinky.Stop()
+}
+
+func (o OracleClient) Prices(ctx context.Context, in *oracletypes.QueryPricesRequest, opts ...grpc.CallOption) (*oracletypes.QueryPricesResponse, error) {
```

Comment:
we should be able to unwrap this content (it's an interface, no)? Try `sdk.UnwrapSDKContent`